app/assets/v2/js/pages/shared_bounty_mutation_estimate_gas.js

```diff
@@ -6,29 +6,19 @@ var gas_amount = function(page_url) {
 
   if (page_url.indexOf('issue/fulfill') != -1) {
     gasLimitEstimate = 207103;
-  }
-  // new fulfill funding page
-  if (page_url.indexOf('/new') != -1) {
+  } else if (page_url.indexOf('/new') != -1) { // new fulfill funding page
     gasLimitEstimate = 318730;
-  }
-  // new fulfill funding page
-  if (page_url.indexOf('issue/increase') != -1) {
+  } else if (page_url.indexOf('issue/increase') != -1) { // new fulfill funding page
     gasLimitEstimate = 56269;
-  }
-  // new process funding page
-  if (page_url.indexOf('issue/accept') != -1) {
+  } else if (page_url.indexOf('issue/accept') != -1) { // new process funding page
     gasLimitEstimate = 103915;
-  }
-  // new kill funding page
-  if (page_url.indexOf('issue/cancel') != -1) {
+  } else if (page_url.indexOf('issue/cancel') != -1) { // new kill funding page
     gasLimitEstimate = 67327;
-  }
-  // tip
-  if (page_url.indexOf('tip/send') != -1) {
+  } else if (page_url.indexOf('issue/payout') != -1) { // bulk payout
+    gasLimitEstimate = 103915;
```

Comment:
should we club this along with `(page_url.indexOf('issue/accept') != -1)` aka [L13](https://github.com/gitcoinco/web/pull/1776/files#diff-15e269c947d7b3b6ece821641f15ada9R13)

@SaptakS @owocki